linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern_empty_string/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Clean sysctl config directories
+rm -rf /usr/lib/sysctl.d/* /run/sysctl.d/* /etc/sysctl.d/*
```

Comment:
What about at least one scenario which would put the option also into sysctl.d

Reply:
Done, added a few more test scenarios in [7358c0f](https://github.com/ComplianceAsCode/content/pull/9396/commits/7358c0fba79d8bd625956763936fada138ff4247)